.travis.yml

```diff
@@ -148,11 +148,14 @@ after_success:
         echo "HOST_CLOUD is NOT set to one of the following values:aws, azure, gce, openstack."
     fi
 
-  # The below script used to be run as post-processor for the Openstack's builder.
+  # The below script for Openstack used to be run as post-processor for the Openstack's builder.
   # In order to avoid running Terraform within Packer, hence getting a cumbersome and tedious log output.
   - >
-    if [ $HOST_CLOUD = 'openstack' ]; then
+    if [ "$HOST_CLOUD" = 'openstack' ]; then
         travis_retry bin/os_pp.sh
+    elif [ "$HOST_CLOUD" = 'aws' ]; then
+        # Script that will copy created AMI from default region into the other used ones
+        bin/aws_pp.sh
```

Comment:
Script added here instead of being run as a standard post-processor for two main reasons:

1. Reduce verbosity (from packer process)
2. Simple solution in order extract the created AMI ID (to be used in the script for copying into other regions)